include/SFML/Audio/Music.hpp

```diff
@@ -69,6 +69,10 @@ class SFML_AUDIO_API Music : public SoundStream
     /// See the documentation of sf::InputSoundFile for the list
     /// of supported formats.
     ///
+    /// \warning Since the music is not loaded at once but rather
+    /// streamed continuously, the file must remain accessible until
+    /// this sf::Music object loads a new music or is destroyed.
```

Comment:
Might be clearer indeed. I initially wrote "this object" (i.e. `*this`) and then added the class name for clarity.